tox.ini

```diff
@@ -16,18 +16,25 @@ deps = -r{toxinidir}/requirements-dev.txt
 [testenv:lint]
 basepython = python3
 commands =
-    flake8 {toxinidir}/src/
+    flake8 {toxinidir}/src/ {toxinidir}/tests/
     pylint {toxinidir}/src/
+    pylint {toxinidir}/tests/ --disable=W0212,R0801,R0201
     mypy {toxinidir}/src/
-    black -l 99 --check {toxinidir}/src/
-    isort --check {toxinidir}/src/
+    black -l 99 --check {toxinidir}/src/ {toxinidir}/tests/
```

Comment:
I would recommend checking the coding guidelines that the k8s team follows before using 99.

Reply:
99 was line-length used in the auto-generated `run_tests.sh` script. I assume that's the standard then for the `Operator` charms.